tools/ci_build/build.py

```diff
@@ -2694,6 +2697,9 @@ def main():
         if args.minimal_build is not None:
             raise BuildError("Python bindings are not supported in a minimal build.")
 
+        if args.enable_dlpack is None:
```

Comment:
Do we need this?

Reply:
I wonder this too. I think the argument can default to true. Just that we don’t want it for the cmake arg such that c++ only compilation will have no dlpack. We only need it for Python was what I meant